doc/man/8/mount.ceph.rst

```diff
@@ -186,6 +217,7 @@ Availability
 refer to the Ceph documentation at http://ceph.com/docs for more
 information.
 
+
```

Comment:
nit: No need for this extra newline.

Reply:
Every section in this doc has 2 newlines, so I followed the patter. :P